data/signal-cli-socket.service

```diff
@@ -5,15 +5,43 @@ After=network-online.target
 Requires=signal-cli-socket.socket
 
 [Service]
-Type=simple
+CapabilityBoundingSet=
 Environment="SIGNAL_CLI_OPTS=-Xms2m"
-ExecStart=%dir%/bin/signal-cli --config /var/lib/signal-cli daemon
-User=signal-cli
+# Update 'ReadWritePaths' if you change the config path here
+ExecStart=/usr/local/bin/signal-cli --config /var/lib/signal-cli daemon
```

Comment:
Done. Although it have to admit initially I got confused when starting the unit because `%dir%` looked like a specifier: https://man.archlinux.org/man/systemd.unit.5#SPECIFIERS